Examples/Algorithms/Vertexing/src/AdaptiveMultiVertexFinderAlgorithm.cpp

```diff
@@ -64,7 +70,8 @@ AdaptiveMultiVertexFinderAlgorithm::AdaptiveMultiVertexFinderAlgorithm(
         return Linearizer(ltConfig,
                           logger().cloneWithSuffix("HelicalTrackLinearizer"));
       }()},
-      m_vertexFinder{makeVertexFinder()} {
+      m_vertexSeeder{makeVertexSeeder()},
```

Comment:
Maybe this should eventually switch to just doing this in `initialize` anyway, to avoid the need to pass members into the functions. But not important.

Reply:
yeah I agree that is not optimal